starter-code/index.js

```diff
@@ -1,11 +1,60 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+    this.sum;
+    this.average;
+  }
+
+  add(x) {
+    this.length++;
+    this.items.push(x);
+    this.items.sort(function(a, b) {
+      return a - b;
+    });
+  }
+
+  get(pos) {
+    return this.items[pos];
```

Comment:
here, you need to do 

```
    return this.items[pos - 1];

```

by doing -1 you fix the off by one error. The index starts at 0. So if the array has 10 items in it, the last index is 9.